libenv/sysinfo.c

```diff
@@ -3784,6 +3899,44 @@ static void SysOsVersionMajor(EvalContext *ctx)
 
 /*****************************************************************************/
 
+static void SysOSVersionMinorPutInt(EvalContext *ctx, int minor, char *tags)
+{
+    char strminor[PRINTSIZE(minor)];
+    xsnprintf(strminor, sizeof(strminor), "%d", minor);
+
+    SysOSVersionMinorPut(ctx, strminor, tags);
+}
+
+static void SysOSVersionMinorPut(EvalContext *ctx, char *strminor, char *tags)
+{
+    EvalContextVariablePutSpecial(ctx,
+        SPECIAL_SCOPE_SYS,
+        "os_version_minor",
+        strminor,
+        CF_DATA_TYPE_STRING,
+        tags
+    );
+}
+
+static void SysOsVersionMinorSetDefault(EvalContext *ctx)
```

Comment:
So this one puts `"Unknown"` if the variable is not already set. Maybe to function name should reflect that. E.g., `SysOsVersionMinorSetUnknownIfUnset()`?